Server/Controllers/MatchController.cs

```diff
@@ -258,7 +258,27 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
 
             return Ok();
         }
-
+        [HttpPost]
+        [Route("patreon_player_update")]
+        public async Task<ActionResult> PatreonPlayerUpdate([FromBody] PatreonPlayerUpdate request)
+        {
+            foreach (var player in request.players)
+            {
+                var ctx = _context.Players.FirstOrDefault(p => p.SteamId.ToString() == player.SteamId);
+                if (ctx == null) continue;
```

Comment:
It should create entities for players that don't exist yet, like in the match end event